configure.ac

```diff
@@ -61,6 +61,9 @@ AC_ARG_ENABLE(terminal-bell,
 AC_ARG_ENABLE(termcap,
    AS_HELP_STRING([--enable-termcap], [Use termcap library to query terminal size.]))
 
+AC_ARG_ENABLE(locale,
```

Comment:
I think I'd prefer for this to be enabled by default, since that's what ppl expect, and detect capability and fall-back to non-utf-8 capable.